src/aap_eda/core/migrations/0057_activation_edited_at.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.2.16 on 2025-01-23 19:06
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("core", "0056_alter_activation_options"),
```

Comment:
where is this 0056 migration file? do you miss it?

Reply:
0056 migration exists in feature/edit-activation branch